### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by Noel-Alex**
Added cogs and updated them to work with the new versions of discord.py (as of 01/07/2024)
Moved summarise.py code to within the summarise cog

### CODE REVIEW

File: cogs/qa_user.py
Comment (sidhant-sriv) [inline cogs/qa_user.py:1]: Was this feature tested?

File: cogs/fun.py
Comment (sidhant-sriv) [inline cogs/fun.py:1]: Remove the reddits stuff unless you can get it to work

File: cogs/qa_user.py
Comment (Noel-Alex) [reply cogs/qa_user.py:1]: yes but only half of them, I thought I'd push it later today, but if you wanna see the code it's here, the music commands as well as the 8ball command aren't working as of now to my knowledge

File: cogs/message.py
Comment (sidhant-sriv) [inline cogs/message.py:1]: This is useless, get rid of it

File: cogs/fun.py
Comment (Noel-Alex) [reply cogs/fun.py:1]: sure then, I thought I'd make tokens a little later and see if they work, I don't have API keys yet

File: cogs/poll.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import discord
+from discord.ext import commands
+
+
+class Poll(commands.Cog):
+    """"""
```
Comment (sidhant-sriv) [inline cogs/poll.py:6]: Refactor and format all files properly

File: cogs/message.py
Comment (Noel-Alex) [reply cogs/message.py:1]: sure

File: cogs/fun.py
Comment (sidhant-sriv) [reply cogs/fun.py:1]: I don't think you can do the reddit API thing anymore, they've discontinued it iirc.

**Comment by sidhant-sriv** [comment]
There are merge conflicts, fix them. 
Remove all unwanted files. 
Add docstrings to explain all the command classes.

File: cogs/poll.py (hunk 1 above)
Comment (Noel-Alex) [reply cogs/poll.py:6]: yes I have a lot of cleaning up and documentation to do

Comment (Noel-Alex) [reply cogs/poll.py:6]: and imma get rid of the reddit stuff as well too thank you

**Pull request closed without merging**